MAUI/Calendar/getting-started.md

```diff
@@ -7,23 +7,45 @@ control: SfCalendar
 documentation: ug
 ---
 
-# Getting started with .NET MAUI Calendar (SfCalendar)
+# Getting Started with the .NET MAUI Calendar
+
 This section explains how to add the [.NET MAUI Calendar](https://www.syncfusion.com/maui-controls/maui-calendar) control. This section covers only the basic features needed to get started with Syncfusion Calendar.
 
 To get start quickly with our .NET MAUI Calendar, you can check the below video.
 
 {% youtube
 "youtube:https://www.youtube.com/watch?v=kfQjKiD9Xas"%}
 
-## Creating an application using the .NET MAUI Calendar
+## Prerequisites
+
+Before proceeding, ensure the following are set up:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
+
+## Step 1: Create a New MAUI Project
+
+### Visual Studio
+
+1. Go to **File > New > Project** and choose the **.NET MAUI App** template.
+2. Name the project and choose a location, then click **Next**.
+3. Select the .NET framework version and click **Create**.
+
+### Visual Studio Code
 
-1. Create a new .NET MAUI application in Visual Studio.
+1. Open the command palette by pressing `Ctrl+Shift+P` and type **.NET:New Project** and enter.
+2. Choose the **.NET MAUI App** template.
+3. Select the project location, type the project name and press enter.
+4. Then choose **Create project.**
 
-2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add `SfCalendar` to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Calendar](https://www.nuget.org/packages/Syncfusion.Maui.Calendar), and then install it.
+## Step 2: Install the Syncfusion MAUI Calendar NuGet Package
 
-3. To initialize the control, import the control namespace `Syncfusion.Maui.Calendar` in XAML or C# code.
+1. In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+2. Search for [Syncfusion.Maui.Calendar](https://www.nuget.org/packages/Syncfusion.Maui.Calendar/) and install the latest version.
+3. Ensure the necessary dependencies are installed correctly, and the project is restored.
 
-4. Initialize `SfCalendar.`
+## Step 3: Register the handler
+
+The [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
 
 {% tabs %}
```

Comment:
Check code snippets forRegister the handler - seems xaml file also added

Reply:
Now changed